### PR TITLE
RavenDB-16493 update docker tags

### PR DESCRIPTION
**Pull request by gregolsky**

### CODE REVIEW

File: docker/common.ps1 [hunk 1]
```diff
@@ -35,15 +35,19 @@ function GetUbuntuImageTags($repo, $version, $arch) {
     switch ($arch) {
         "x64" { 
             return @(
-                "$($repo):$($version)-ubuntu.20.04-x64",
-                "$($repo):5.2-ubuntu-latest"
+                "$($repo):latest",
+                "$($repo):latest-lts",
+                "$($repo):ubuntu-latest",
```
Comment (ppekrol) [inline docker/common.ps1:40]: should we consider `ubuntu-latest-lts` ?

File: docker/publish-nanoserver.ps1 [hunk 2]
```diff
@@ -42,7 +42,7 @@ function PushImages($imageTags) {
 }
 function GetImageTags($repo, $version) {
         return @(
-            #"$($repo):windows-nanoserver-latest",
+            "$($repo):windows-nanoserver-latest",
```
Comment (ppekrol) [inline docker/publish-nanoserver.ps1]: Should we consider `windows-nanoserver-latest-lts` ?

Comment (gregolsky) [reply docker/publish-nanoserver.ps1]: No harm in adding it I guess.
BTW this reminds we need to update the docker readme to reflect these changes.

Also I thought we could shorten it to `windows-latest` and `windows-latest-lts` maybe?

File: docker/common.ps1 (hunk 1 above)
Comment (gregolsky) [reply docker/common.ps1:40]: How about `ubuntu-lts`?

**Review by ppekrol — APPROVED** [review]

**Pull request merged**